apps/pano/src/models/comment.server.ts

```diff
@@ -23,22 +23,44 @@ export type CommentWithUpvotes = Prisma.CommentGetPayload<
 
 export type Comment = CommentWithOwner & CommentWithUpvotes;
 
+export const normalizeComment = (model: Comment) => {
+  if (model.deletedAt !== null) {
```

Comment:
let's not modify the input comment, instead clone and modify the clone and return it.